packages/components/src/Spinner.purs

```diff
@@ -55,5 +55,18 @@ loadingSpinner =
         ]
     }
 
+loadingSpinnerWithMessage :: String -> JSX
```

Comment:
The alignment of the spinner and the message breaks a bit here: the spinner seems to be aligned left, although it should be centered. As the `.spinner--loading-spinner` is a [grid](https://css-tricks.com/snippets/css/complete-guide-grid/), we can adjust the alignment by setting [justify-items](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items) to `center`.

I don't think the `align-content` and `justify-content` attributes, which we have defined in that class, are actually useful anymore.

Reply:
I didn't touch spinner's styles but instead added `.spinner--loading-spinner` to the div containing the message and made it a sibling of the div containing the spinner. With this, both are justified to the center and the message is on top of the spinner. I'll change it if you have a better idea about how it should look like, this was what I was aiming for initially but I used a too short string for testing.

Reply:
Yep, this works! 👍